ext/api_util.cpp

```diff
@@ -15,6 +15,18 @@
 
 using namespace boost::python;
 
+#if _MSC_VER > 1800
+namespace boost
+{
+	template <>
+	Tango::ApiUtil const volatile * get_pointer<class Tango::ApiUtil const volatile >(
+		class Tango::ApiUtil const volatile *c)
+	{
+		return c;
+	}
+}
+#endif
```

Comment:
Maybe this code block can be refactored to be included only once?

Reply:
Surely, but I don't know where could be the best place to add it.